backend/model/model.go

```diff
@@ -2831,62 +2925,76 @@ func (obj *Alert) sendSlackAlert(ctx context.Context, db *gorm.DB, alertID int,
 		for index, addr := range input.RelatedFields {
 			relatedFormattedFields = relatedFormattedFields + fmt.Sprintf("%d. *%s*: `%s`\n", index+1, addr.Name, addr.Value)
 		}
-		// construct Slack message
-		previewText = fmt.Sprintf("Highlight: Track Properties Alert (%s)", identifier)
-		textBlock = slack.NewTextBlockObject(slack.MarkdownType, fmt.Sprintf("*Highlight Track Properties Alert (`%s`):*\n\n", identifier), false, false)
-		messageBlock = append(messageBlock, slack.NewTextBlockObject(slack.MarkdownType, fmt.Sprintf("*Matched Track Properties:*\n%+v", matchedFormattedFields), false, false))
-		messageBlock = append(messageBlock, slack.NewTextBlockObject(slack.MarkdownType, fmt.Sprintf("*Related Track Properties:*\n%+v", relatedFormattedFields), false, false))
-		blockSet = append(blockSet, slack.NewSectionBlock(textBlock, messageBlock, nil))
-		blockSet = append(blockSet, slack.NewDividerBlock())
-		msg.Blocks = &slack.Blocks{BlockSet: blockSet}
+
+		// header
+		headerBlock := slack.NewTextBlockObject(slack.MarkdownType, "*Track Properties Alert*", false, false)
+		headerBlockSet = append(headerBlockSet, slack.NewSectionBlock(headerBlock, nil, nil))
+
+		// body
+		var attributeBlocks []*slack.TextBlockObject
+		attributeBlocks = append(attributeBlocks, slack.NewTextBlockObject(slack.MarkdownType, fmt.Sprintf("*Matched Track Properties*\n%+v", matchedFormattedFields), false, false))
+		attributeBlocks = append(attributeBlocks, slack.NewTextBlockObject(slack.MarkdownType, fmt.Sprintf("*Related Track Properties*\n%+v", relatedFormattedFields), false, false))
+
+		bodyBlockSet = append(bodyBlockSet, slack.NewSectionBlock(sessionBlock, attributeBlocks, nil))
 	case AlertType.USER_PROPERTIES:
```

Comment:
![Screenshot 2023-10-09 at 7 55 13 PM](https://github.com/highlight/highlight/assets/17744174/3b305b56-dbc4-43df-8ef6-f7d3abe9f262)

Reply:
should this be `identified_email: true` instead of the json array syntax?

Reply:
<img width="676" alt="Screenshot 2023-10-10 at 9 56 08 AM" src="https://github.com/highlight/highlight/assets/17744174/92e5504e-6697-4482-9145-3970ed56673a">